build/stf-run-ci/tasks/setup_base.yml

```diff
@@ -40,7 +40,6 @@
         name: amq7-cert-manager
         source: redhat-operators
         sourceNamespace: openshift-marketplace
-        startingCSV: amq7-cert-manager.v1.0.0
```

Comment:
Just start at the latest version as `startingCSV` will still upgrade through to the latest CSV version in the channel. I'm not aware of ability to pin. The `startingCSV` is probably only useful to us in upgrade testing scenarios I guess?